src/core/tsi/local_transport_security.cc

```diff
@@ -127,32 +129,50 @@ static tsi_result handshaker_result_create_zero_copy_grpc_protector(
   return ok;
 }
 
+static tsi_result handshaker_result_get_unused_bytes(
+    const tsi_handshaker_result* self, const unsigned char** bytes,
+    size_t* bytes_size) {
```

Comment:
nit. How about doing sanity check on `self`, `bytes` and `bytes_size` to make sure they are not `nullptr`, and returns `TSI_INVALID_ARGUMENT` if the check fails?

Reply:
Done.